core/src/main/java/com/novoda/noplayer/PlayerView.java

```diff
@@ -1,6 +1,8 @@
 package com.novoda.noplayer;
 
 import android.view.View;
+
+import com.google.android.exoplayer2.ui.AspectRatioFrameLayout;
```

Comment:
I think we need to remove ExoPlayer dependencies from this public interface, you can have a factory or something like that for your enum, that's private cc @Mecharyry

Reply:
this interface is at the same level as `NoPlayerView` that implements it and uses `AspectRatioFrameLayout` internally. To me it seems like using the constants from `AspectRatioFrameLayout` here is the same internal usage

Reply:
🤔 I take your point @JozefCeluch, `NoPlayerView` is poorly named, it's really `ExoPlayerView` 😂 really the mapping should live in the implementation of the interface and not in the `interface` itself. That way clients that implement the `PlayerView` are free to change the `AspectRatioFrameLayout` for whatever they like.

Reply:
@Mecharyry @zegnus what do you think about this https://github.com/novoda/no-player/pull/281/commits/c0a40a2cf26d6c9c72a20c77847df427ee7e2829?